src/client/cmd/launch.cpp

```diff
@@ -160,7 +161,27 @@ mp::ParseCode cmd::Launch::parse_args(mp::ArgParser* parser)
     {
         try
         {
-            auto node = YAML::LoadFile(parser->value(cloudInitOption).toStdString());
+            YAML::Node node;
+            const QString& cloudInitFile = parser->value(cloudInitOption);
+            if (cloudInitFile == "-")
```

Comment:
We should check if we are running in a TTY before going any further here.  There is already a convenience function defined in `include/multipass/cli/client_platform.h` and implemented already for each platform.

Reply:
Good point, done